src/Renderer/c3DEngine.js

```diff
@@ -38,11 +38,13 @@ function c3DEngine(rendererOrDiv) {
         this.renderer.setSize(this.width, this.height);
     }.bind(this);
 
-    //
+    var canvas = document.createElement('canvas');
+    canvas.tabIndex = -1;
+    canvas.addEventListener('click', () => canvas.focus());
```

Comment:
Also, the canvas doesn't need to be created at all if `renderer` is supplied by the user.